addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/fuzz/WebSocketFuzzerTask.java

```diff
@@ -44,15 +44,15 @@ protected WebSocketFuzzer getParent() {
 
     @Override
     protected void runImpl(WebSocketMessageDTO message, List<Object> payloads) {
-        getParent().preProcessMessage(getId(), message, payloads);
+        getParent().preProcessMessage(getIndex(), message, payloads);
```

Comment:
These should kept as `getId()`.
It was accidentally changed in https://github.com/davy320/zap-extensions/pull/1/commits/44b0495bcf08eabd4a78559196001c602ba52161#diff-738e9549bce92f185c31726eea4a4c30R39

Reply:
Well, I intentionally changed them, so maybe it would better to say "ignorantly changed"... ;-)

I originally pulled your repo and built only the fuzz add-on, but then noticed it was a lower version than upstream's fuzz module. I then merged zaproxy/zap-extensions, which was clean except for the top of the changelog, but then fuzz failed to build. After diffing zaproxy and davy320 versions, I noted zaproxy had performed a large number of `getId()`-to-`getIndex()` transformations, so I more or less mindlessly made those to fuzz also until I was able to achieve a successful build (of fuzz alone, i.e. `gradlew :addOns:fuzz:build`)

The result of that is what I've been using internally in my application testing.

What I understand you to be asking for is a revert of most of those renames. I'm fine with the concept, but since empirically most/all of them are required to reconcile with current zaproxy, what should we be doing instead?

Thanks for the guidance!

Reply:
The rename pointed earlier is what caused most/all of the renames, the ones strictly needed are in `UserHttpFuzzerMessageProcessorUIHandler` (`Context#getId()` was changed to `Context#getIndex()`).